etc/profile-m-z/spotify.profile

```diff
@@ -22,6 +22,7 @@ mkdir ${HOME}/.config/spotify
 mkdir ${HOME}/.local/share/spotify
 whitelist ${HOME}/.cache/spotify
 whitelist ${HOME}/.config/spotify
+whitelist ${HOME}/.config/spotify-adblock
```

Comment:
blacklist+noblacklist?

Reply:
@rusty-snake [on Dec 2](https://github.com/netblue30/firejail/pull/5498#discussion_r1038331113):

> blacklist+noblacklist?

Good catch, I forgot about that.

Done.